csharp/test/Microsoft.ML.OnnxRuntime.Tests/OnnxMl.cs

```diff
@@ -332,7 +332,7 @@ public sealed partial class AttributeProto : pb::IMessage<AttributeProto> {
     /// implementations needed to use has_field heuristics to determine
     /// which value field was in use.  For IR_VERSION 0.0.2 or later, this
     /// field MUST be set and match the f|i|s|t|... field in use.  This
-    /// change was made to accomodate proto3 implementations.
+    /// change was made to accommodate proto3 implementations.
```

Comment:
> /// change was made to accommodate proto3 implementations. [](start = 3, length = 59)

Looks great, but this file is auto generated by protoc compiler for C#. We may want to edit the proto file from which this was generated.

Reply:
This is the updated auto generation from the latest onnx proto file. Technically the file probably shouldn't be checked in at all, but it was so I'm not sure if there's some historical reason for that.  Can look into that later.